package student_rolands_gudels.lesson_12_exceptions.level_2_intern;

import teacher.annotations.CodeReview;

@CodeReview(approved = true)
public class Task_17 {
/*    Почему класс AccessDeniedException в нашем случае
    лучше наследовать от Exception, а не от RuntimeException?

    class AccessDeniedException extends Exception {

   Потому - что в RuntimeException могут быть сгенерированы, но не пойманы во время выполнения метода.

    }*/
}
